File: lab6/C#/SmallLetters.cs
using System;
using RyadkiLibrary;

namespace SmallLettersLibrary
{
    public class SmallLetters:Ryadki
    {
        public SmallLetters(string line) : base(line)
        { }
        public override int RyadokLength()
        {
            return str.Length;
        }
        override public void InsertSymbols()
        {
            string newstr = "";
            for (int i = 1; i < str.Length; i += 2)
            {
                newstr = str.Insert(i, "\\");
                str = newstr;
            }
        }
    }
}

File: lab6/C++/SmallLetters.cpp
#pragma once
#include "Header.h"
#include "Ryadki.cpp"

class SmallLetters :public Ryadki
{
public:
    SmallLetters(string line) : Ryadki(line)
    { }
    int RyadokLength() override
    {
        return str.length();
    }
    void InsertSymbols() override
    {
        string newstr = "";
        for (int i = 1; i < str.length(); i += 2)
        {
            newstr = str.insert(i, "\\");
            str = newstr;
        }
    }
};